src/EnergyPlus/UnitarySystem.cc

```diff
@@ -0,0 +1,12162 @@
+                this->designCoolingCapacity = initUnitarySystemsQActual;
+
+            } // from IF(UnitarySystem(UnitarySysNum)%CoolingCoilType_Num == Coil_CoolingWater .OR. Coil_CoolingWaterDetailed
+            if (this->heatingCoilType_Num == DataHVACGlobals::Coil_HeatingWater) {
+
+                //     set water-side mass flow rates
+                Real64 mdot = this->maxHeatCoilFluidFlow;
+                PlantUtilities::SetComponentFlowRate(mdot,
+                                                     this->heatCoilFluidInletNode,
+                                                     this->heatCoilFluidOutletNodeNum,
+                                                     this->heatCoilLoopNum,
+                                                     this->heatCoilLoopSide,
+                                                     this->heatCoilBranchNum,
+                                                     this->heatCoilCompNum);
+                //     simulate water coil to find operating capacity
+                WaterCoils::SimulateWaterCoilComponents(this->heatingCoilName, FirstHVACIteration, this->heatingCoilIndex, initUnitarySystemsQActual);
+                this->designHeatingCapacity = initUnitarySystemsQActual;
+
+            } // from IF(UnitarySystem(UnitarySysNum)%HeatingCoilType_Num == Coil_HeatingWater) THEN
+
+            if (this->heatingCoilType_Num == DataHVACGlobals::Coil_HeatingSteam) {
+
+                //     set water-side mass flow rates
+                Real64 mdot = this->maxHeatCoilFluidFlow;
+                PlantUtilities::SetComponentFlowRate(mdot,
+                                                     this->heatCoilFluidInletNode,
+                                                     this->heatCoilFluidOutletNodeNum,
+                                                     this->heatCoilLoopNum,
+                                                     this->heatCoilLoopSide,
+                                                     this->heatCoilBranchNum,
+                                                     this->heatCoilCompNum);
+                //     simulate steam coil to find operating capacity
+                SteamCoils::SimulateSteamCoilComponents(
+                    this->heatingCoilName,
+                    FirstHVACIteration,
+                    this->heatingCoilIndex,
+                    1.0,
+                    initUnitarySystemsQActual); // QCoilReq, simulate any load > 0 to get max capacity of steam coil
+                this->designHeatingCapacity = SteamCoils::GetCoilCapacity(
+                    DataHVACGlobals::cAllCoilTypes(this->heatingCoilType_Num), this->heatingCoilName, initUnitarySystemsErrorsFound);
+
+            } // from IF(UnitarySystem(UnitarySysNum)%HeatingCoilType_Num == Coil_HeatingSteam) THEN
+            if (this->suppHeatCoilType_Num == DataHVACGlobals::Coil_HeatingWater) {
+
+                //     set steam-side mass flow rates
+  
```

Comment:
@Myoldmopar I need your help here. Put a break point at 9150 of UnitarySystems.cc and show me how to call SolveRoot. Par is now a std::vector and the compiler is telling me to use a reference to the function. I don't understand. Uncomment 9151 to test.

Reply:
I hope it was obvious that running the unit test gets you here.